pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -224,6 +224,10 @@ func (m *MultishareOpsManager) startInstanceWorkflow(ctx context.Context, w *Wor
 	}
 	switch w.opType {
 	case util.InstanceCreate:
+		if m.msControllerServer != nil && !m.msControllerServer.featureMaxSharePerInstance {
+			// MaxShareCount field set to 0, defaults to 10 shares in Filestore
+			w.instance.MaxShareCount = 0
```

Comment:
this seem to be extra step since MaxShareCount is already set to 0 before?

Reply:
this is an extra safeguard to set the value to 0 if feature disabled. thoughts?

Reply:
Ack. The generateNewMultishareInstance checks for the feature flag before setting the MaxShareCount. Removing this extra check here